docs/migration-progress.md

```diff
@@ -0,0 +1,127 @@
+# Migration progress
+
+UCX tracks migration progress of _business_ resources: workspace objects that contribute to business value.
+(The term "business resource" comes from the UCX team and is **not** Databricks terminology.) We identified the
+following business resources:
+
+| Business resource          | Motivation                                                                                                                 |
+|----------------------------|----------------------------------------------------------------------------------------------------------------------------|
+| Dashboard                  | Dashboards visualize data models supporting business processes                                                             |
+| Job                        | Jobs create data models supporting business process - not exclusively data models used by dashboards                       |
+| Delta Live Table pipelines | Delta Live Table pipelines create data models supporting business process - not exclusively data models used by dashboards |
+
+Furthermore, UCX tracks migration of the following Hive and workspace objects:
+
+| Hive or workspace object           |
+|------------------------------------|
+| Tables and view (Hive data object) |
+| Grant                              |
+| User defined function (UDF)        |
+| Cluster                            |
+| Cluster policies                   |
+
+See the [resource index](#resource-index) for more details on the above objects.
+
+## Usage
+
+Use the migration progress through the [migration progress dashboard](../README.md#dashboards) after running the
+[(experimental) migration progress workflow](../README.md#experimental-migration-progress-workflow).
+
+### Failures
+
+A [key historical attribute](#historical) in migration progress are the `failures` that show the incompatibility issues
+with Unity Catalog. By resolving the failures for an object, UCX flags that object to be Unity Compatible. Thus,
+for Hive data objects, this means that the objects are migrated to Unity Catalog.
+
+### Owner
+
```

Comment:
Define ownership. Explain the logic.

Reply:
Separate issue: https://github.com/databrickslabs/ucx/issues/3067